types/svelte-leafletjs/index.d.ts

```diff
@@ -76,9 +76,12 @@ export namespace LeafletContext {
     }
 }
 
+type KeyEvents = Array<keyof MapEvents>;
+
 export class LeafletMap extends SvelteComponentTyped<
     {
         options?: L.MapOptions;
+	    events?: KeyEvents;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/svelte-leafletjs/index.d.ts;
  Line: 84;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.